Lib/fontgoggles/mac/mainWindow.py

```diff
@@ -303,9 +303,10 @@ def setupSidebarGroup(self):
                                                        showHiddenAxes=self.project.uiSettings.showHiddenAxes)
 
         self.variationsGroup.instances = LabeledView(
-            (0, 0, 0, 1),  # gets updated as axes are read
+            # Position and instances gets updated as fonts are read
+            (0, 0, 0, 20),
```

Comment:
What does the `x, y, w, h` destructuring in LabledView do, if only checking for `h > 0`  — should it be removed?

Reply:
I really don't understand the `posSize` ... dug down into `VanillaBase` but can't figure it out. I tried `(10, y, -10, 0)` (updated also in the `_updateSidebarItems` call) like the sliders, but that just results in weird overlaps and cut offs.